v22.1/take-full-and-incremental-backups.md

```diff
@@ -150,36 +154,36 @@ Periodically run the [`BACKUP`][backup] command to take a full backup of your cl
 ~~~ sql
-> BACKUP INTO '{collection location}';
+> BACKUP INTO '{collectionURI}';
 ~~~
 
-Then, create nightly incremental backups based off of the full backups you've already created. To append an incremental backup to the most recent full backup created in the [collection's](#backup-collections) storage location, use the `LATEST` syntax:
+Then, create nightly incremental backups based off of the full backups you've already created. To append an incremental backup to the most recent full backup created at the [collection's](#backup-collections) URI, use the `LATEST` syntax:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> BACKUP INTO LATEST IN '{collection location}';
+> BACKUP INTO LATEST IN '{collectionURI}';
 ~~~
 
-<span class="version-tag">New in v22.1:</span> This will add the incremental backup to the default `/incrementals` directory at the root of the backup collection's directory. With incremental backups in the `/incrementals` directory, you can apply different lifecycle policies from cloud storage providers to the `/incrementals` directory as needed.
+<span class="version-tag">New in v22.1:</span> This will add the incremental backup to the default `/incrementals` directory at the root of the backup collection's directory. With incremental backups in the `/incrementals` directory, you can apply different lifecycle/retention policies from cloud storage providers to the `/incrementals` directory as needed.
 
 {{site.data.alerts.callout_info}}
-If there are existing incremental backups in the storage location taken with v21.2 or earlier, new incremental backups will add to the existing incremental directory, (e.g., `{collection location}/{subdirectory}`). This follows how [incremental backups](take-full-and-incremental-backups.html#take-an-incremental-backup) were stored by default in v21.2 and earlier. To back up using the prior behavior, see this [example](#backup-earlier-behavior) for details on using the `incremental_location` option to achieve this.
+If there are existing incremental backups in the storage location taken with v21.2 or earlier, new incremental backups will add to the existing incremental directory, (e.g., `{collectionURI}/{subdirectory}`). This follows how [incremental backups](take-full-and-incremental-backups.html#take-an-incremental-backup) were stored by default in v21.2 and earlier. To back up using the prior behavior, see this [example](#backup-earlier-behavior) for details on using the `incremental_location` option to achieve this.
 {{site.data.alerts.end}}
 
 If it's ever necessary, you can then use the [`RESTORE`][restore] statement to restore your cluster, database(s), and/or table(s). Restoring from incremental backups requires previous full and incremental backups.
 
-To restore from an incremental backup, stored in the default `/incrementals` collection directory, run:
+To restore from an incremental backup, stored in the default `/incrementals` collection subdirectory, run:
```

Comment:
I think the phrase "restore from an incremental backup" is slightly misleading, as it implies we only restore the incremental backup and not the whole chain. 

why  not say something like:  "restore from the latest backup in the collection." and the for the next example, just say "restore from a specific backup in the collection"